cmd/admission-webhook/main.go

```diff
@@ -51,10 +56,32 @@ func main() {
 	}
 	version.LogVersionInfo()
 
-	podAh := &webhook.PodAdmissionHook{
+	ah := &webhook.AdmissionHook{
 		ExtraServiceAccounts:     extraServiceAccounts,
 		EvictRegionLeaderTimeout: evictRegionLeaderTimeout,
 	}
-	stsAh := &webhook.StatefulSetAdmissionHook{}
-	cmd.RunAdmissionServer(podAh, stsAh)
+	run(flagset, ah)
+}
+
+func run(flagset *flag.FlagSet, admissionHooks ...apiserver.AdmissionHook) {
+
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
+		runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
why is this necessary?

Reply:
+1. If we need to configure GOMAXPROCS, we can just use uber's `automaxprocs` package.

Reply:
This is introduced in the orgin framework by [cmd.RunAdmissionServer](https://github.com/openshift/generic-admission-server/blob/92cbaa8d51b1b9fa33a7eda76683b3482590e534/pkg/cmd/cmd.go#L28).
Because I want to import my extra flagset, I copied this function like [quack](https://github.com/pusher/quack/blob/5d7531613d6bf969b9018fde40a5cffb7d3efd7e/cmd/quack/main.go#L34).

Reply:
I think Go runtime will use NumCPU if the GOMAXPROCS=0 automatically.

Reply:
updated, removing this logic.